Inc/DirectXMath.h

```diff
@@ -411,7 +411,7 @@ namespace DirectX
 #elif defined(_XM_SSE_INTRINSICS_)
         inline operator __m128i() const noexcept { return _mm_castps_si128(v); }
         inline operator __m128d() const noexcept { return _mm_castps_pd(v); }
-#elif defined(_XM_ARM_NEON_INTRINSICS_) && (defined(__GNUC__) || defined(_ARM64_DISTINCT_NEON_TYPES))
+#elif defined(_XM_ARM_NEON_INTRINSICS_) && defined(__GNUC__) || defined(_ARM64_DISTINCT_NEON_TYPES))
```

Comment:
I think it would be clearer here to add parenthesis to make it clear. I assume this is supposed to be:

```
#elif defined(_XM_ARM_NEON_INTRINSICS_) && (defined(__GNUC__) || defined(_ARM64_DISTINCT_NEON_TYPES))
```

Reply:
Oops, this was an unintended change, a consequence of doing things in a hurry. I have removed it and verified that all other changes are intended.